test/suites/Tests-8-ECMA.js

```diff
@@ -66,11 +66,13 @@ define(['pumascript', 'esprima'], function(puma, esprima) {
      });
 
      test("configurable", function(){
```

Comment:
Its one of the things i groomed unrelated to the do while visitar.
*It was failing.
*The error message is "implementation dependent" and as such shouldn't be used on an equality test.
*Pruned some unnecessary code for consiness.